student-bios/podnesTaF.md

```diff
@@ -0,0 +1,32 @@
+# Hi there!
+## I'm Oleksii Pidnebesnyi
+
+<img src="https://th.bing.com/th/id/OIP.p5aepZh8OpbpDE8YlmVtWgHaHa?pid=ImgDet&rs=1" width="100" height="100">
```

Comment:
Hi, check this post here to understand how you should place an image in a markdown file https://stackoverflow.com/questions/41604263/how-do-i-display-local-image-in-markdown